### PR TITLE
Use common intro portal

### PR DESCRIPTION
**Pull request by swemail**
Make use of a common `IntroPortalComponent` and resolve Live/Async specific things in container components

### CODE REVIEW

**Review by transient-jonas — APPROVED** [review]
LGTM! Nice work!

File: client/src/lib/session/hooks/useLogInSessionMetricEvents.ts [hunk 1]
```diff
@@ -15,37 +15,65 @@ type AllowedSharingEvents =
...
+        metrics.logEvent(event, {
+          'Sharing Session ID': asyncSession.id,
+          'Sharing Session Type': asyncSession.type,
+          'Sharing Session Start Time': asyncSession.startTime,
+          'Sharing Session Duration': dayjs().diff(
+            asyncSession.startTime,
+            'seconds',
+          ),
+          'Exercise ID': asyncSession.contentId,
+          Host: false,
```
Comment (gewfy) [inline client/src/lib/session/hooks/useLogInSessionMetricEvents.ts:61]: This is a matter of definition 😄  Also I'm thinking of adding a prop that differentiate these two. Like `Sharing Session Mode`: `live/async`. Nothing you need to change. Just thinking loud here... in text... 🤔

**Review by gewfy — APPROVED** [review]
👌

File: client/src/lib/session/hooks/useLogInSessionMetricEvents.ts (hunk 1 above)
Comment (swemail) [reply client/src/lib/session/hooks/useLogInSessionMetricEvents.ts:61]: Thought that we still pass sessionType so from that we can deduce that Host is not important for async sessions. But room for improvement, definitely

**Pull request merged**